src/app/shared/broadcaster.service.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import {Subject} from "rxjs/Subject";
+import {Observable} from "rxjs/Observable";
+import "rxjs/add/operator/filter";
+import "rxjs/add/operator/map";
+
+import { Injectable } from "@angular/core";
+
+interface IBroadcastEvent {
+  key: any;
+  data?: any;
+}
+
+@Injectable()
+export class Broadcaster {
```

Comment:
I know this is a pretty trivial class and that we have a separate Trello card to improve doc, but when adding new stuff it would be nice to add at least top-level class documentation to it. Something like (sure you can come up with a better description):

```typescript
/**
 * Class implementing a broadcast mechanism equivalent to the one available in Angular 1
*/